src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
As per the comments above, you can replace this block with this suggestion

```suggestion
  const men = people.filter((person) => {
    const isMan = person.sex === "m";
    const isDiedInThisCentury = century
      ? Math.ceil(person.died / 100) === century
      : true;

    return isMan && isDiedInThisCentury;
  });
```